dist/windows/clementine.nsi.in

```diff
@@ -337,12 +374,9 @@ Section "Clementine" Clementine
   File "Qt5Network.dll"
   File "Qt5OpenGL.dll"
   File "Qt5Sql.dll"
-  File "Qt5Svg.dll"
```

Comment:
We have some svg icons this will probably break

Reply:
Nothing links to Qt5Svg.dll with the current included plugins. In case of svg icons we need imageformats/qsvg.dll which is missing both from the dependencies makefile and the nsi. I can add it.

Reply:
Fixed